src/routes/views/components/charts/trendChart/trendChart.tsx

```diff
@@ -49,6 +49,7 @@ interface TrendChartOwnProps {
   showSupplementaryLabel?: boolean; // Show supplementary cost labels
   showUsageLegendLabel?: boolean; // The cost legend label is shown by default
   title?: string;
+  chartName: string;
```

Comment:
@dlabrecq - apparently this is the culprit for `intl` being required prop for `ReportSummaryComponet`... making `chartName` optional doesn't matter.. any idea why this would be?